config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -270,20 +270,20 @@ teams:
   cluster-api-release-team:
     description: Release team of Cluster API
     members:
-    - alexander-demicev
     - aniruddha2000
     - CecileRobertMichon
-    - cpanato
-    - dntosas
+    - chiukapoor
+    - chrischdi
     - furkatgofurov7
+    - g-gaston
     - hackeramitkumar
     - joekr
+    - johannesfrey
+    - killianmuldoon
     - nawazkh
-    - oscr
-    - sayantani11
+    - prajyot-parab
```

Comment:
not sure if it matters but @Prajyot-Parab's username is upper cased

Reply:
Looks like for VibhorChinda we used the actual casing, so we should change it (does this apply to anyone else as well?

Reply:
I checked everyone else, this is the only one that was inconsistent

Reply:
good call. I've check all the other names for casing changes.